### PR TITLE
Resolved outdated node-fetch dependency

### PR DESCRIPTION
**Pull request by lukemarkwordtlibertyits**
The okta-sdk had an outdated isomorphic-fetch dependency. This dependency had an outdated node-fetch dependency that caused dependabot to report a minor security vunerability. Okta-SDK's latest patch updated the isomorphic-fetch dependency version to the latest version, which resolved its node-fetch issue.
[Regression Tests](https://github.com/department-of-veterans-affairs/vets-saml-proxy/files/5403530/API2418_Tests__RegressionTests-V2.docx)

### CODE REVIEW

**Review by crolarlibertyva — CHANGES_REQUESTED** [review]
I updated this dependency during API-2399.  This could be the reason for the conflict.

File: oauth-proxy/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/department-of-veterans-affairs/oauth-proxy#readme",
   "dependencies": {
-    "@okta/okta-sdk-nodejs": "^3.3.1",
+    "@okta/okta-sdk-nodejs": "^4.2.0",
```
Comment (crolarlibertyva) [inline oauth-proxy/package.json:25]: I updated this dependency during API-2399.  This could be the reason for the conflict.

Comment (lukemarkwordtlibertyits) [reply oauth-proxy/package.json:25]: yes, just resolved.

**Comment by lukemarkwordtlibertyits** [comment]
The rebase merged in a weird way. I feel more comfortable creating a new branch for this PR.

**Pull request closed without merging**